Rakefile

```diff
@@ -14,3 +13,13 @@ desc "Run rubocop"
 task :rubocop do
   sh "rubocop"
 end
+
+desc "Bundle all gemfiles"
+task :bundle_all do
+  Bundler.with_original_env do
+    system("which matching_bundle 2>&1") || abort("gem install matching_bundle")
```

Comment:
Why not make it a dev dependency?

Reply:
I cannot be a dev dependency since then it would have to be bundled to be used and the issue this is trying to fix is that bundling does not work :)